src/test/java/io/xstefank/wildfly/bot/utils/Constants.java

```diff
@@ -0,0 +1,13 @@
+package io.xstefank.wildfly.bot.utils;
+
+public class Constants {
+
+    public static final String XSTEFANK_REPO = "xstefank/wildfly";
+    public static final String PR_JSON = "pr-template.json";
+    public static final String INVALID_TITLE = "Invalid title";
+    public static final String INVALID_DESCRIPTION = "Invalid description";
+    public static final String INVALID_COMMIT_MESSAGE = "Invalid commit message";
+    public static final String VALID_TITLE = "[WFLY-123] Valid title";
+    public static final String VALID_DESCRIPTION = "https://issues.redhat.com/browse/WFLY-123";
+    public static final String VALID_COMMIT_MESSAGE = "[WFLY-123] Valid commit message";
```

Comment:
Why you don't put default valid values into default JSON template?

Reply:
In order to clarify in the tests that test the valid values that the JSON indeed contains those valid values. Removing those values changes nothing, but I wanted to emphasize that a test is using valid values especially when you go through the code in 3 years